specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json

```diff
@@ -0,0 +1,1762 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-10-01",
+    "title": "AdvisorManagementClient",
+    "description": "REST APIs for Azure Advisor"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Advisor/metadata/{name}": {
+      "get": {
+        "tags": [
+          "Metadata"
+        ],
+        "summary": "Gets the metadata entity.",
+        "operationId": "RecommendationMetadata_Get",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of metadata entity.",
```

Comment:
could you add the pattern for restriction?

Reply:
@sjanamma : Hello ! Am not sure about the ask here . Will reach out to you offline wrt to resolving this

Reply:
Thanks for the review - Advisor team has this comment taken as PBI to include the change suggested here : https://msazure.visualstudio.com/One/_backlogs/backlog/AzureAdvisor/Features/?workitem=17048171 and will introduce the change needed by the comment in the next api-version increment of advisor.